spec/models/manager_refresh/save_inventory/acyclic_graph_of_inventory_collections_spec.rb

```diff
@@ -596,7 +596,7 @@
             :model_class => Hardware,
             :parent      => @ems,
             :association => :hardwares,
-            :manager_ref => [:vm_or_template]
+            :manager_ref => [:virtualization_type]
```

Comment:
sure.  is there a better place to test that (with literal nil, independent of `.find`)?